### ISSUE TITLE
Color doesn't work on Windows

### ISSUE THREAD

**Issue by AceofSpades5757** [oberblastmeister/trash-cli#113]
```
trashy 2.0.0

OS Name:                   Microsoft Windows 10 Business
OS Version:                10.0.19045 N/A Build 19045
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
```

Color doesn't appear to work on Windows machines.

![image](https://github.com/oberblastmeister/trashy/assets/10341888/093c8fd4-0b93-4e04-b2d5-d2b9c3b8b7a6)

**Comment by deadcoder0904** [comment]
can confirm.

**Comment by GeorgeTR1** [comment]
I'm not sure this counts as a bug, since the colors are based on the colors that `ls` uses. Windows doesn't have `ls`, and `dir` doesn't use colors, so to me this behavior makes sense for Windows

**Comment by AceofSpades5757** [comment]
Makes sense. May want to put a note/disclaimer under the features then. 

Any chance it'd work with ls installed on Windows and configured for colored output?

**Comment by GeorgeTR1** [comment]
No, unfortunately. It explicitly checks if the os is linux:
https://github.com/oberblastmeister/trashy/blob/7c48827e55bca5a3188d3de44afda3028837b34b/src/app/command/list.rs#L233-L238
In my opinion that's ok, though, most people probably don't have `ls` installed on Windows, and even if they do, the colored output is a lot less useful. The concept of "marking as executable" doesn't exist, files without extensions are pretty rare, so determining what's a directory is easier, and symlinks are also a lot rarer. 

It would be ideal I think to just make a note of this in the information, and probably just entirely remove the `-c` option when trashy is built for Windows.

I also initially was a bit confused that `-c` only affects the `ls` style colors in the list output, it doesn't affect color elsewhere, like in the error/help messages. But, when I piped the output of the help into a file there weren't any escape codes, so maybe clap already handles that automatically? Or is that something that bash does? I don't know. Anyway, that's also something the help could be more clear about.

If anyone fixes any of these things, though, I don't think it will be me, I don't have any experience with rust, I just know how to grep around some source code a bit. Unfortunately this project seems abandoned, though, so I don't know that anyone else will either. I think that's a shame, though, the idea of a trash has been around over 40 years, and yet most people working in the terminal are still just permanently deleting files with `rm`. Then, the only cli trash utility that seems to have achieved some ubiquity is written in Python of all things

If you're interested in a cli utility to move files to the recycle bin on Windows, you can check out [recycle-bin](https://github.com/GeorgeTR1/recycle-bin/). It's under 100 lines of C that compiles to a 4 kB exe. All it does is move files and directories to the recycle bin, assuming that you then use the standard Windows gui when you want to restore something

**Comment by AceofSpades5757** [comment]
I like `trashy` just fine, but thanks for the recommendation. And thanks for the work you did on it. The project is perfectly fine as is.
